scripts/api_rst_coverage.py

```diff
@@ -13,6 +13,30 @@ def class_name_sort_key(x):
         else:
```

Comment:
can you add a note about using this?

ideally we would also add these scripts in a not in the contributing docs.

Reply:
can you update for this?